src/auth/index.ts

```diff
@@ -21,4 +30,31 @@ const setupAuth = password => {
   }
 }
 
-export { setupAuth, isCorrectCredentials }
+// This will be changed into setupAuth at a PR later
+const setupAccessToken = (req, res, next): void => {
+  const { adminPass } = req.cookies
+  const { access = '', authorization = '' } = req.headers
+  req.user = {
+    isAdmin: false,
+    isUser: false
+  }
+  if (adminPass === hashPass(pass)) {
+    req.user.isAdmin = true
```

Comment:
admins should be users too, because of the `validUi` check below: `if (!req.user.isUser) return res.redirect('/login')`

this code block should be above, right below: `  const { adminPass } = req.cookies`